src/react-ruler/Ruler.tsx

```diff
@@ -84,42 +87,36 @@ export default class Ruler extends React.PureComponent<RulerProps> implements Ru
     const {
       unit,
       type,
-      backgroundColor,
-      lineColor,
-      textColor,
       textBackgroundColor,
       direction,
       negativeRuler = true,
       segment = 10,
       textFormat,
+      theme,
       range = [-Infinity, Infinity],
       rangeBackgroundColor,
     } = props as Required<RulerProps>;
+    const { backgroundColor, lineColor, textColor } = theme === 'dark' ? DARK_THEME : LIGHT_THEME;
     const width = this.width;
     const height = this.height;
     const state = this.state;
     state.scrollPos = scrollPos;
     const context = this.canvasContext;
     const isHorizontal = type === 'horizontal';
     const isNegative = negativeRuler !== false;
-    const font = props.font || '10px sans-serif';
+    const font = props.font || '10px Inter';
```

Comment:
you probably want to add some safe and fallback fonts in case Inter is missing for some reason